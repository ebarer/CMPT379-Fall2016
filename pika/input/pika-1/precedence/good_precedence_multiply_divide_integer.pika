exec {

  print 86 * 70 / 79, _n_.
  print 14 * 95 / 12, _n_.
  print 33 * 48 / 12, _n_.
  print 97 * 85 / 70, _n_.
  print 34 * 17 / 53, _n_.
  print 41 * 54 / 68, _n_.
  print 47 * 55 / 48, _n_.
  print 34 * 8 / 33, _n_.
  print 39 * 61 / 23, _n_.
  print 64 * 79 / 76, _n_.
  print 28 * 41 / 87, _n_.
  print 28 * 63 / 47, _n_.

  print 38 / 59 * 57, _n_.
  print 56 / 34 * 6, _n_.
  print 85 / 50 * 7, _n_.
  print 32 / 11 * 38, _n_.
  print 51 / 79 * 72, _n_.
  print 86 / 82 * 34, _n_.
  print 9 / 23 * 29, _n_.
  print 43 / 16 * 59, _n_.
  print 62 / 24 * 96, _n_.
  print 72 / 37 * 56, _n_.
  print 63 / 10 * 5, _n_.
  print 91 / 34 * 23, _n_.

  print 37 * 84 / 42 * 53, _n_.
  print 73 * 56 / 4 * 20, _n_.
  print 20 * 4 / 98 * 66, _n_.
  print 84 * 34 / 50 * 84, _n_.
  print 73 * 9 / 31 * 84, _n_.
  print 71 * 85 / 74 * 49, _n_.
  print 70 * 32 / 93 * 69, _n_.
  print 33 * 49 / 31 * 76, _n_.
  print 4 * 4 / 19 * 38, _n_.
  print 93 * 49 / 59 * 79, _n_.
  print 8 * 91 / 1 * 28, _n_.
  print 96 * 58 / 71 * 54, _n_.

  print 52 / 65 * 46 / 43, _n_.
  print 51 / 60 * 30 / 58, _n_.
  print 22 / 82 * 62 / 33, _n_.
  print 3 / 98 * 78 / 54, _n_.
  print 30 / 88 * 75 / 55, _n_.
  print 8 / 59 * 45 / 94, _n_.
  print 93 / 19 * 59 / 26, _n_.
  print 45 / 2 * 53 / 18, _n_.
  print 18 / 46 * 17 / 68, _n_.
  print 30 / 83 * 19 / 58, _n_.
  print 27 / 76 * 36 / 54, _n_.
  print 60 / 92 * 10 / 65, _n_.

}

