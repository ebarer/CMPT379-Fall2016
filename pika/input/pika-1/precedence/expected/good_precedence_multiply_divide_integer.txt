76
110
132
117
10
32
53
8
103
66
13
37
0
6
7
76
0
34
0
118
192
56
30
46
3922
20440
0
4788
1764
3969
1656
3952
0
6083
20384
4212
0
0
0
0
0
0
9
64
0
0
0
0
